conda/cli/conda_argparse.py

```diff
@@ -125,20 +129,26 @@ def do_call(args, parser):
     module = import_module(relative_mod, __name__.rsplit(".", 1)[0])
 
     command = relative_mod.replace(".main_", "")
-    _run_pre_command_hooks(command, args)
+    _run_command_hooks("pre", command, args)
+    result = getattr(module, func_name)(args, parser)
+    _run_command_hooks("post", command, args, result)
 
-    return getattr(module, func_name)(args, parser)
+    return result
```

Comment:
same as above?

Reply:
I'll change the function signature (`args` -> `arguments`) and add a type annotation to make it super clear.